independent-publisher-connectors/Connpass/apiDefinition.swagger.json

```diff
@@ -0,0 +1,322 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "connpass",
+    "description": "Using the connpass API, you can search and collect information about IT workshops and events.",
+    "version": "1.0",
+    "contact": {
+      "name": "Miyake Hideo",
+      "email": "h.miyake@hotmail.com"
+    }
+  },
+  "host": "connpass.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/v1/event/": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "results_start": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "results_start",

```

Comment:
@Miyake-Mito please look through all descriptions and titles and make sure that they look like valid sentences. More information about English strings can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
Thank you for reviewing this.
All descriptions and titles have been checked and corrected.